Makefile

```diff
@@ -66,6 +66,9 @@ build-gateway:
 	cd ./services/gateway && docker build --tag=lisk/service_gateway ./
 
 build-template:
+	cd ./services/export && docker build --tag=lisk/service_export ./
+
+build-export:
 	cd ./services/template && docker build --tag=lisk/service_template ./
```

Comment:
Please cherry-pick the commit `4efb5d4b4b812c747fad6006b146d00cf358f4a4` to fix the commands (build-export currently builds template and vice-versa)

Reply:
Done